src/pages/new/index.md

```diff
@@ -0,0 +1,62 @@
+---
+keywords:
+  - New UXP Features
+  - InDesign UXP Features
+---
+# What's New
+This section contains details about the latest UXP features available in InDesign.
```

Comment:
Please follow the changelog format specified here https://keepachangelog.com/en/1.0.0/.

The idea is to provide a preview of what's changing on this page. And provide a link to the details residing in recipes/guides.

Reply:
Another example https://developer.adobe.com/photoshop/uxp/2022/ps_reference/changelog/

Reply:
Done